### PR TITLE
ROC-29 registration be

### PR DESCRIPTION
**Pull request by squaill**
refactored and moved us to sqlalchemy. I suggest perusing `https://flask-sqlalchemy.palletsprojects.com/en/3.0.x/quickstart/#configure-the-extension` if you want to learn more about the sql alchemy ORM

### CODE REVIEW

File: rockr/auth0/auth0_api_wrapper.py [hunk 1]
```diff
@@ -13,8 +16,8 @@ def _get_api_token(self):
     def get_user_role(self, user_id):
         token = self._get_api_token()
         resp = self.http.request("GET",
-                                 f"{self.settings.AUTH0_URL}/users/{user_id}/roles",
+                                 f"{self.settings.API_TOKEN}/users/{user_id}/roles",
```
Comment (LarrySchultheis) [inline rockr/auth0/auth0_api_wrapper.py]: Y u change

Comment (squaill) [reply rockr/auth0/auth0_api_wrapper.py]: I don't remember changing that tbh

Comment (LarrySchultheis) [reply rockr/auth0/auth0_api_wrapper.py]: Well fix it

**Review by LarrySchultheis — APPROVED** [review]
Merging. Refactors us to use SQL Alchemy.

**Pull request merged**